### DISCUSSION TITLE
https://blog.3place.cn/3place/sszs/SSZS001.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#104] [General]
# https://blog.3place.cn/3place/sszs/SSZS001.html

https://blog.3place.cn/3place/sszs/SSZS001.html

**Comment by picklecai** [comment]
他的失败，也应该使我们对他表示更深厚的恭敬与同情，因为偌大的世界之中，只有他有这信心，冒了绝大的危险，费了无数的麻烦，牺牲了一切平凡的安逸，牺牲了家庭的亲谊和人间的名誉，去追求，去试验一个“梦想之神圣境界”，而终于免不了惨酷的失败；也不完全是他的人生观的失败。他的失败是因为他的信仰太单纯了，而这个现实世界太复杂了，他的单纯的信仰禁不起这个现实世界的摧毁；

**Comment by picklecai** [comment]
他的同情，他的鼓舞，早又在别的园里种出了无数的可爱的小树，开出了无数可爱的鲜花。他自己的歌唱有一个时代是几乎消沉了；但他的歌声引起了他的园地外无数的歌喉，嘹亮的唱，哀怨的唱，美丽的唱。
